src/sentry/rules/conditions/event_frequency.py

```diff
@@ -545,6 +550,42 @@ def query_hook(
 
         return 0
 
+    def batch_query_hook(
+        self, group_ids: Sequence[int], start: datetime, end: datetime, environment_id: str
+    ) -> dict[int, int]:
+        batch_percents: dict[int, int] = defaultdict(int)
+        groups = Group.objects.filter(id__in=group_ids)
+        project_id = groups[0].project.id
+        session_count_last_hour = self.get_session_count(project_id, environment_id, start, end)
+
+        if session_count_last_hour >= MIN_SESSIONS_TO_FIRE:
+            interval_in_minutes = (
+                percent_intervals[self.get_option("interval")][1].total_seconds() // 60
+            )
+            avg_sessions_in_interval = session_count_last_hour / (60 / interval_in_minutes)
+            error_issues = [
```

Comment:
Just calling out that I'm fairly certain this condition only applies to error issues, so I just pulled those one so we're not attempting to query snuba on other issues that are definitely going to return 0 and is useless data.

Reply:
I think this should be sufficient, I guess we could pull them out before calling this but it doesn't really matter.